src/oc_erchef/schema/t/test_create_and_update_users.sql

```diff
@@ -0,0 +1,216 @@
+CREATE OR REPLACE FUNCTION test_zzzzcreate_and_update_users()
+RETURNS SETOF TEXT
+LANGUAGE plpgsql
+AS $$
+BEGIN
+
+-- v0 add_user tests --
+RETURN QUERY SELECT is(add_user('11111111111111111111111111111111',
+                                '22222222222222222222222222222222',
+                                'aaaa',
```

Comment:
For these dummy values, I'd recommend having them describe what they are as much as you can given any constraints that exist on the data.

So if "aaaa" is a username, use "test_user" instead, etc.

Reply:
updated!

Reply:
done!